test/statement_test.ts

```diff
@@ -30,9 +30,9 @@ describe('statements', () => {
   });
   it('translates try/catch', () => {
     expectTranslate('try {} catch(e) {} finally {}')
-        .to.equal(' try { } catch ( e ) { } finally { }');
+        .to.equal(' try { } catch ( e , e_stack ) { } finally { }');
```

Comment:
How 'ideally' is that? It's challenging to look ahead in that way.

On Fri, May 8, 2015 at 2:55 PM Miško Hevery <notifications@github.com>
wrote:

> In test/statement_test.ts
> <https://github.com/angular/ts2dart/pull/153#discussion_r29980110>:
>
> > @@ -30,9 +30,9 @@ describe('statements', () => {
> >    });
> >    it('translates try/catch', () => {
> >      expectTranslate('try {} catch(e) {} finally {}')
> > -        .to.equal(' try { } catch ( e ) { } finally { }');
> > +        .to.equal(' try { } catch ( e , e_stack ) { } finally { }');
>
> ideally we should only get the stack if we need it, as it is expensive.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/angular/ts2dart/pull/153/files#r29980110>.
>